python/example/ring.py

```diff
@@ -48,18 +48,32 @@ def event_generators(self, gid):
 context = arbor.context(threads=4, gpu_id=None)
 print(context)
 
+meters = arbor.meter_manager()
+print(meters)
+
+meters.start(context)
+
 recipe = ring_recipe(100)
 print(recipe)
 
+meters.checkpoint('recipe create', context)
```

Comment:
It is best to avoid using spaces in checkpoint names, so that the text output is easier to parse (e.g. using grep or awk).
For example use `recipe-create`, `load-balance`, `simulation-init`, etc.

Reply:
Done.